inc/styles.php

```diff
@@ -526,7 +526,8 @@ static function filter_css_object( $css, $panels_data, $post_id, $layout ) {
 			}
 
 			// Add in flexbox alignment to the main row element
-			if ( ! empty( $row['style']['cell_alignment'] ) ) {
+			$flexbox_align_styles = array('auto','flex-start', 'center', 'flex-end', 'stretch');
+			if ( ! empty( $row['style']['cell_alignment'] ) && in_array($row['style']['cell_alignment'], $flexbox_align_styles ) ) {
```

Comment:
I like Flexbox and we use all the existing options, but we are adding additional (not flexbox related) options.

The problem, as I see it, is that vertical alignment is now limited to flexbox and it is not extendable beyond it. 
This was just an idea how to resolve it, but there might be better solution?

I would appreciate if you find another solution to add custom options to vertical alignment without creating errors in CSS output would be OK too.